### ISSUE TITLE
Fork closing PRs

### ISSUE THREAD

**Issue by kevindmoore** [ForkIssues/Tracker#1382]
I'm not sure how Fork could do this but somehow Fork closed several PRs when I was trying to delete local branches.

Steps:
1. Switch to master branch
2. Make sure I've pulled the latest code
3. Delete older branch from PR.
4. Fork gives me a warning that the branch hasn't been merged (but it has)
5. Delete the branch anyway
6. Do this for about 4 branches

I then get asked why I closed several PRs on github (which I haven't touched). While I can't see how Fork could have done this, this was the only git operation I performed.

**Comment by DanPristupov** [comment]
No, Fork, can't do that.

Does GitHub show details about why (and when) the PRs were closed?

**Comment by kevindmoore** [comment]
So, I looked at the PR that was closed and I believe it was closed to a
branch being deleted.
I've been having problems with Fork in that it doesn't seem to know that a
branch has been already merged and asks if I am sure I want to delete a
branch that hasn't been merged to master.
I'm wondering if Fork gets confused about which branch it is pointing to on
the server.
Luckily we were able to restore the branch on github.
Thanks,
Kevin Moore
